tests/test_tiff.py

```diff
@@ -125,7 +127,7 @@ def test_geotiff_write(tmpdir):
     assert np.allclose(numpy_image, data)
 
     writer = TiffWriter(filename)
-    writer.initialize((3, 5, 1), numpy_image.dtype)
+    writer.initialize((WIDTH, HEIGHT, 1), numpy_image.dtype)
```

Comment:
Why is it usually height width but here is width height?

Reply:
I mostly did not change function signatures so they are still width, height.  If you prefer I can change them to make things more consistent.

Reply:
Yeah, I think we should. If the tile shape is (height, width) I think it will be less confusing if everything else is too. What do you think? I guess this is also a question of what people who deal with satellite imagery would be most used to. So you are the best equipped to answer this.

Reply:
Images are usually indexed by row, column but some image libraries do it the other way.  I went ahead and switched everything over to row,col with the exception of our Rectangle class.